cli.js

```diff
@@ -221,11 +202,4 @@ function init(args, options) {
 sudoBlock();
 updateNotifier({pkg: pkg}).notify();
 
-if (process.stdin.isTTY) {
-	init(args, options);
-} else {
-	getStdin(function (data) {
-		[].push.apply(args, data.trim().split(/\r?\n/));
-		init(args, options);
-	});
-}
+init(subarg(cli.input, options)._, cli.flags);
```

Comment:
No, `crop` and `delay` can be passed into each subcommand. See: https://github.com/sindresorhus/pageres-cli/blob/eabc83255c92a7b2ec7b4769b493883512811f37/cli.js#L57